### PR TITLE
 Bug 1837934: Fix default value different from state

### PR DESCRIPTION
**Pull request by bipuladh**
Backing Store creation default value set in both the states.

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx [hunk 1]
```diff
@@ -198,6 +198,12 @@ const PVCType: React.FC<PVCTypeProps> = ({ state, dispatch }) => {
     TiB: 'TiB',
   };
 
+  // Noobaa expected Ti console standrad is to show TiB
+  const unitConverter = {
+    GiB: 'Gi',
+    TiB: 'Ti',
+  };
```
Comment (gnehapk) [inline frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx:205]: This should be moved to constants file.

Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx:205]: This is already merged. See the .. diff.

**Pull request merged**